test/test_no_regression.py

```diff
@@ -0,0 +1,39 @@
+# coding=utf-8
+from walter import project
+import pandas,os
+from walter.data_access import get_data_dir
+from pathlib2 import Path
+import numpy.testing as np
+
+def test_same_result_zerolight():
+    #assert : verify if the same wheat field have the same results for the same chosen parameters.
+
+    p = project.Project(name='same') # Create the simulation directory
+    params = p.csv_parameters('sim_scheme_test.csv')[0] # recovery of parameters
+    params.update(dict(nbj=165)) #Adding parameters to vary
```

Comment:
Given what I know about WALTer, I think this test should be done on a whole simulation (i.e nbj = 325 and not 165)